### PR TITLE
IJ-97 + IJ-91 

### PR DESCRIPTION
**Pull request by IeuanSkinner**
I've added in the ability for admins to pause team member accounts via a button on the team members index. This button has a confirmation warning explaining that the team member will be locked out of the system whilst paused and the team member is informed when they attempt to access the system why that can't.

I've also gone ahead and removed the box-shadow properties from all buttons whilst here which is IJ-91.

Screenshot 2021-04-07 at 10 42 57

Screenshot 2021-04-07 at 10 43 14

Screenshot 2021-04-07 at 10 43 54

### CODE REVIEW

**Comment by IeuanSkinner** [comment]
There's a bug in here with the flash messages not appearing that has been fixed in the IJ-98 branch.

File: app/controllers/team_members/team_members_controller.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   class TeamMembersController < AdminApplicationController
-    before_action :team_member, only: %i[show approve_team_member toggle_admin]
-    before_action :unapproved_team_members, :approved_team_members, only: :index
+    before_action :team_member, except: :index
 
     # GET /team_members
     def index
+      @unapproved_team_members = TeamMember.unapproved.order(created_at: :desc)
+      @approved_team_members = TeamMember.approved.order(created_at: :desc)
+
```
Comment (phil-6) [inline app/controllers/team_members/team_members_controller.rb:10]: Extra empty line

**Review by phil-6 — APPROVED** [review]
Overall LGTM.
Only question is around semantics paused / unpaused or active / suspended

File: app/controllers/team_members/team_members_controller.rb (hunk 1 above)
Comment (IeuanSkinner) [reply app/controllers/team_members/team_members_controller.rb:10]: This is intentional, an extra space between the instance variable assignments and controller render action can improve readability.

**Comment by IeuanSkinner** [comment]
> Only question is around semantics paused / unpaused or active / suspended

The Jira ticket specifically says pause/paused.

**Pull request merged**